recipes-graphics/wayland/weston_10.0.0.bbappend

```diff
@@ -9,7 +9,10 @@ SRC_URI:append:tegra = " file://0001-Drop-DRM-version-check-in-meson.build.patch
 DEPENDS:append:tegra = " egl-wayland"
 PACKAGE_ARCH:tegra = "${TEGRA_PKGARCH}"
 
-RDEPENDS:${PN}:append:tegra = " egl-wayland egl-gbm tegra-udrm-probeconf"
+RDEPENDS:${PN}:append:tegra = " egl-wayland egl-gbm"
 
 # wlshell is used by nvidia gstreamer plugins
 EXTRA_OEMESON:append:tegra = " -Ddeprecated-wl-shell=true"
+
+RRECOMMENDS:${PN}:append:tegra194 = " kernel-module-tegra-udrm"
+RRECOMMENDS:${PN}:append:tegra234 = " kernel-module-nvidia-drm"
```

Comment:
These SoC-specific appends would mean having to set PACKAGE_ARCH to make this package SoC-specific, too.

I *think* it's safe to just use `:tegra` here and mention both of the kernel modules, since RRECOMMENDS is a weak dependency.

Reply:
I will add `PACKAGE_ARCH` 
regarding the `tegra194` vs `tegra234`, I wanted to be explicit on the `RRECOMMENDS` 
I will change it to tegra and add both kernel modules in the same list

Reply:
If you do what I suggest, you **do not** need to set PACKAGE_ARCH.  It's one or the other.